spaceship.zsh

```diff
@@ -70,12 +74,24 @@ spaceship_user() {
 spaceship_host() {
   if [[ -n $SSH_CONNECTION ]]; then
     echo -n "$(spaceship_user)"
-    echo -n " %B${SPACESHIP_PREFIX_USER}%b "
+
```

Comment:
Actually, this code can be rewritten simpler. You can simplify this:

```sh
if [[ ${SPACESHIP_PREFIX_SHOW} == true ]]; then 
  echo -n " %B${SPACESHIP_PREFIX_SSH_DIR}%b "
fi
```

to this:

```sh
[[ ${SPACESHIP_PREFIX_SHOW} == true ]] && echo -n " %B${SPACESHIP_PREFIX_SSH_DIR}%b "
```

I'm sure this small change is gonna make code more clean.